operator/apis/lifecycle/v1alpha3/keptntask_types.go

```diff
@@ -168,18 +168,18 @@ func (t KeptnTask) SetSpanAttributes(span trace.Span) {
 
 func (t KeptnTask) CreateKeptnLabels() map[string]string {
 	if t.Spec.Workload != "" {
-		return map[string]string{
+		return common.MergeMaps(map[string]string{
```

Comment:
I would swap the order of the parameters. Since the second win in the case of a common key, I would prefer having our hardcoded value being there